app/views/dashboard/subsidies/_form.html.haml

```diff
@@ -44,7 +44,7 @@
             .form-group
               = form.label "percent_subsidy", t(:dashboard)[:subsidies][:form][:percent], class: "col-sm-4 control-label"
               .input-group.col-sm-7
-                = form.text_field "percent_subsidy", { class: 'form-control', value: display_as_percent(subsidy.percent_subsidy) }
+                = form.text_field "percent_subsidy", { class: 'form-control', value: display_as_percent(subsidy.percent_subsidy || subsidy.default_percentage) }
```

Comment:
extract subsidy.percent_subsidy || subsidy.default_percentage in to subsidy method and reuse in other places